jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/RxSource.java

```diff
@@ -20,5 +20,6 @@
   SAT, // XM satelite
   RV,  // for BSM rx
   SNMP,// for SRM payload from backend/ODE
-  NA, unknown
+  NA,  // Not applicable (for example, Distress Notification or Driver Allert 
```

Comment:
Spelling in the comment
`// Not applicable (for example, Distress Notification or Driver Allert `
--> `// Not applicable (for example, Distress Notification or Driver Alert)`